.changeset/bitter-actors-matter.md

```diff
@@ -0,0 +1,7 @@
+---
+'@mysten/seal': patch
+---
+
+Force scalar encoding in BLS to big-endian since versions >=1.9.6 of noble/curves changed the default encoding to little-endian.
+Encryptions created by previous versions of Seal SDK and with noble/curves versions >=1.9.6 might fail to `decrypt` with the default call arguments. In case you need to decrypt such ciphertexts, set  `checkShareConsistency=fallse` and `checkLENonce=true` on `DecryptOptions`.
```

Comment:
```suggestion
Encryptions created by previous versions of Seal SDK and with noble/curves versions >=1.9.6 might fail to `decrypt` with the default call arguments. In case you need to decrypt such ciphertexts, set  `checkShareConsistency=false` and `checkLEEncoding=true` on `DecryptOptions`.
```